controller/controller_server/messages.py

```diff
@@ -7,6 +7,7 @@
 # validation error messages
 invalid_secret_message = "invalid secret was passed"
 secret_missing_message = 'secret is missing'
+volume_context_missing_message = 'volume_context is missing'
```

Comment:
I thought this field is optional

Reply:
yep, it was not in use, I removed it

Reply:
then please add(/open a ticket to add) "[allow-global-unused-variables](https://github.com/IBM/ibm-block-csi-driver/pull/158#discussion_r432831053)=no" to our pylint configuration, to prevent it from re-happening.

Reply:
created  CSI-1660